perception/lidar_apollo_instance_segmentation/src/node.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 TierIV
+// Copyright 2020 TIER IV, Inc.
```

Comment:
Just a minor point, but I think 2020 is still Tier IV, not TIER IV.

Reply:
@yukkysaito Is it okay to update the year?

Reply:
I think either would be fine.

Reply:
You can update the year, but there is no obligation to update it.

Reply:
@yukkysaito I updated the years at https://github.com/autowarefoundation/autoware.universe/pull/3320/commits/55e530872f16755120f57aedbb70ecec60c401db.

Reply:
@wep21 Sorry, my instruction was unclear. When you update the year, you need to keep the original year. So it would be `2020-2023`.

Reply:
@kenji-miyake I see. I fixed them at https://github.com/autowarefoundation/autoware.universe/pull/3320/commits/55e530872f16755120f57aedbb70ecec60c401db.